wrapper/src/test/java/integration/host/TestEnvironment.java

```diff
@@ -471,15 +473,41 @@ private static String getAuroraDbEngine(TestEnvironmentRequest request) {
     }
   }
 
-  private static String getAuroraDbEngineVersion(TestEnvironmentRequest request) {
+  private static String getAuroraDbEngineVersion(
+      TestEnvironment env,
```

Comment:
"env" contains all that the method needs. I think a single parameter would be enough.